app/pages/lab/subject-sets/IndexedSubjectSet.jsx

```diff
@@ -0,0 +1,32 @@
+import isAdmin from '../../../lib/is-admin';
+
+export default function IndexedSubjectSet({ subjectSet }) {
+  function updateMetadata(event) {
+    const data = new FormData(event.target);
+    const indexFields = data.get('indexFields');
+    subjectSet.update({ 'metadata.indexFields': indexFields });
+    subjectSet.save();
+    event.preventDefault();
+  }
+
+  const readOnly = !isAdmin()
+  return (
+    <>
+      <h4>Index settings</h4>
+      <p>Subjects in this set are indexed and searchable.</p>
+      <p><a href={`https://subject-set-search-api.zooniverse.org/subjects/${subjectSet.id}`}>View the index.</a></p>
```

Comment:
Do we want project owners to be able to see this link?

Reply:
I say keep the link. 👌  If a project team has indexed subject sets enabled, then they're likely prepared for a bit of system complexity.